src/next/jdbc/sql/builder.clj

```diff
@@ -165,6 +164,12 @@
           (if batch? identity cat)
           rows)))
 
+(comment
```

Comment:
@seancorfield oh, you pushed comment block. lol. Is it intended?

Reply:
See https://github.com/seancorfield/next-jdbc/commit/454fa21f421b76f18eca5eff1f1d54a520fdf9ba#commitcomment-89072119